src/trello_conn.py

```diff
@@ -0,0 +1,56 @@
+from trello import TrelloClient
+import requests
+
+
+def card_already_in_trello(card_id):
+    with open("trello_cards") as file:
+        lines = file.readlines()
+    if f"{card_id}\n" in lines:
+        return True
+    else:
+        with open('trello_cards', 'a') as file:
+            file.write(f'{card_id}\n')
+        return False
```

Comment:
This reads the whole file every time a single card is check, please read it only once. Also make this part of a class.